### DISCUSSION TITLE
How to disable the animation

### DISCUSSION THREAD

**Discussion by don4ecap** [flameshot-org/flameshot#2297] [Q&A]
I get glitches in the animation, the opening animation is okay but when drawing on the screenshot it is very annoying. Is there a way to turn it off? Thank you

**Comment by mmahmoudian** [comment]
We don't have animation in Flameshot! Please post a video to clarify what you mean by animation.

**Reply by don4ecap** [reply]
yes,, I'm not sure too this is an animation, maybe glitches or something like that

**Reply by don4ecap** [reply]
Wait,, I will send you the video from my screen later
